generators/server/templates/gradle/profile_dev.gradle.ejs

```diff
@@ -45,15 +45,16 @@ bootRun {
 
 <%_ if (!skipClient) { _%>
 task webpackBuildDev(type: <%= _.upperFirst(clientPackageManager) %>Task) {
-    inputs.files(fileTree("<%= CLIENT_MAIN_SRC_DIR %>"))
+    inputs.file("package-lock.json")
```

Comment:
Good catch to put `package-lock` here

Reply:
Thanks. Found it [here](https://guides.gradle.org/running-webpack-with-gradle/#step_3_declare_task_inputs_and_outputs) actually